schema/workflow.json

```
@@ -339,6 +339,10 @@
           "minimum": 0,
           "minLength": 0,
           "description": "Maximum number of retry attempts. Value of 0 means no retries are performed"
+        },
+        "jitter": {
+          "type": "string",
+          "description": "Maximum random amount of time added or subtracted from the delay between each retry (ISO 8601 repeatable format)"
```

I wonder if we can reword this a little, saying this parameter represents a "random amount" might mean that its like a random number, different for each instance of this particular workflow, but it's not.

---

Well, the amount of time that gets added is the random part, so maybe we could move the "random" word a bit further down, like:

```suggestion
          "description": "Maximum amount of random time added or subtracted from the delay between each retry (ISO 8601 repeatable format)"
```

I think this way it's harder to interpret that the maximum amount is random. Wdyt?

---

+1